### PR TITLE
[FA-943] request client token

### PR DESCRIPTION
**Pull request by Olson3R**
# Title
[FA-943] request client token
## What?
Added
- get_client_token to get an access token using a CloudVault application client id and secret.
- create_alert mutation
## Why?
We need a better strategy for getting access tokens where a user isn't involved.
## QA Strategy
- [ ] Merged latest master
- [ ] Updated version number
- [ ] All private git packages are at their newest version in both *Pipfile* and *environment.yml*
- [ ] If changed *Pipfile*, updated *Pipfile.lock*

### CODE REVIEW

File: sentera/api.py [hunk 1]
```diff
@@ -8,15 +8,43 @@
 
 
 def _run_sentera_query(query, token):
```
Comment (elliott-imhoff) [inline sentera/api.py]: Maybe have a ``url_param`` passed in that defaults to ``"https://api.sentera.com/graphql"`` but can be configured to a different endpoint if desired.

File: sentera/api.py [hunk 2]
```diff
@@ -8,15 +8,43 @@
...
-        url="https://api.sentera.com/graphql", json=query, headers=headers
+        url="http://localhost:3001/graphql", json=query, headers=headers
+        # url="https://api.sentera.com/graphql", json=query, headers=headers
     )
     if request.status_code != 200:
         raise Exception("Request Failed {}. {}".format(request.status_code, query))
 
     return request.json()
 
+def create_alert(field_sentera_id, name, message, token):
```
Comment (elliott-imhoff) [inline sentera/api.py]: Will need to add a doc string to pass Travis

**Comment by elliott-imhoff** [comment]
@Olson3R To pass Travis, install ``pre-commit`` (See the installation instructions in **py-image-metadata-parser** for a good example), and make sure all files pass using the command 

``pre-commit run --all-files``

Then commit and push whatever reformatting pre-commit does.  I hope this makes sense.

**Comment by Olson3R** [comment]
@elliott-imhoff This is mainly my quick and dirty proof of concept for our plan to operationalize the analytics alerting models. We plan to shortly improve them.
Thank you for your suggestions though.

**Comment by Olson3R** [comment]
Replaced by individual PRs

**Pull request closed without merging**